src/cordova/build.action.mjs

```diff
@@ -30,6 +30,12 @@ export async function main(...parameters) {
 
   await runAction('cordova/setup', ...parameters);
 
+  if (platform === 'osx' && buildMode === 'release') {
+    throw new Error(
+      'Production MacOS builds currently must be done in XCode due to a cordova issue. Please open platforms/osx/Outline.xcodeproj to continue.'
```

Comment:
Perhaps we should document here what the issue is, so we don't forget. I think it was the build config that was setting the wrong CODE_SIGN_IDENTITY?

Reply:
Sure, will add a comment.